pynitrokey/fido2/client.py

```diff
@@ -375,9 +362,7 @@ def is_solo_bootloader(
             pass
         return False
 
-    def enter_st_dfu(
-        self,
-    ):
+    def enter_st_dfu(self) -> None:
```

Comment:
I think this is about the trailing comma.  For `black` that indicates that you want the separated elements on separate lines.  If you remove the comma, the lines are collapsed if possible.  This is called [“magic trailing comma”](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma).

Reply:
I knew this would come up, so I intentionally de-blacked them again. Sure black is black, in detail better to be discussed with some beer. But this single most-exaggerated case makes me *all-kind-of-things*. Just take a short look at these 4 lines of "code". There is next to no  information transferred, already with the single line (to be precise: callable's name), please please please, don't increase the amount of empty space and non-existing information even further... let's please use `--skip-magic-trailing-comma`, can we pleaaaassse :question: :question:

Reply:
I agree - no need for unnecessary whitespace. The whole point though was to establish a single code format and then adjust it as we go.
@robin-nitrokey anything against trailing comma?
@daringer can we make a separate PR with that, with just an updated black pass? Let's keep it as it is for now (so please format it with the black, with the old settings)

Reply:
But can’t we solve this by just removing the comma?

Reply:
I’d rather not disable that rule completely because a) I don’t see the point here – why is there a comma anyway?  b) there are legitimate use cases where black would put the elements on the same line but the developer decides that having them on separate lines is more readable.

Reply:
`make black fix` left the function headers alone, as long as there is no trailing comma... good enough for me - since day one I dislike the vertical function header rule in pep8 most, but there I've reached acceptance already. for `self` only methods this will be hard to achieve ;) good middle way so far I would say...

Reply:
Sounds good. Let's clear that comma and continue we the default rule set.